CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+# v68.0.0 (_2020-11-18_)
+
+[Full Changelog](https://github.com/mozilla/application-services/compare/v67.0.0...v68.0.0)
+
+### What's Changed
+
+- The bundled version of Glean has been updated to v33.4.0.
+  (as part of [#3724](https://github.com/mozilla/application-services/pull/3724))
+
+## Autofill
+
+### What's New
+
+The autofill component has a first cut at Kotlin bindings and is now bundled in
+the full megazord.
```

Comment:
As a minor nit, with a major version number I was expecting to see at least one "breaking change" called out here. Is there a breaking change that's been missed, or do the changes feel big enough to make a major version bump?

(This should absolutely not be a blocker to proceeding with this release, just something for us to consider).

Reply:
No, this absolutely should be 67.1.0 as I didn't find any breaking changes in the change log (or in the PRs that have been merged since the last release was cut). I initially thought there were breaking changes as some of the smoke tests didn't pass locally. I'll make this change--thanks!